styles/style.css

```diff
@@ -0,0 +1,420 @@
+    margin-bottom: 1rem;
+    justify-content: flex-end;
+    align-content: center;
+}
+
+.pay p {
+    margin-top: 2rem;
+}
+
+.pay img {
+    margin-right: 1rem;
+}
+
+nav {
+    background-color: lightgray;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+    font-weight: bold;
+}
+
+nav li,
+footer li {
+    list-style-type: none;
+    padding: 1rem;
+
+}
+
+nav ul,
+footer ul {
+    display: flex;
+    flex-direction: row;
+
+}
+
+h2 {
+    font-size: 1.5rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+}
+
+
+button {
+    text-align: center;
+    color: white;
+    background-color: rgb(18, 146, 18);
+    font-weight: bold;
+}
+
+.dogs button {
+    display: block;
+    margin: 1rem auto;
+    width: 80%;
+    border: solid 1px rgb(18, 146, 18);
+
+}
+
+.more-dogs {
+    background-color: lightgrey;
+    color: navy;
+    font-weight: bold;
+    border: solid 1px lightgrey;
+    display: block;
+    width: 30%;
+    margin: 2rem auto 0 auto;
+    padding-top: .5rem;
+    padding-bottom: .5rem;
+
+
+}
+
+.submit button {
+    color: navy;
+    font-weight: bold;
+    background-color: lightgrey;
+    width: 10%;
+}
+
+.dogs img {
+    width: 100%;
+}
+
+.grey-border{
+    border: lightgrey solid 3px;
+}
+
+.green-border, .chosen {
+    border: rgb(18, 146, 18) solid 3px;
+}
+
+.no-border {
+    border: none;
+}
+
+.dogs h3,
+.dogs h4,
+.dogs p,
+.dogs button {
+    padding-right: .5rem;
+    padding-left: .5rem;
+    padding-bottom: .5rem;
+}
+
+.adopt {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-bottom: 5rem;
+    padding-top: 3rem;
+    width: 100%;
+}
+
+.adopt img {
+    width: 100%;
+}
+
+.dogs {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(14rem, 32%));
+    grid-column-gap: 1%;
+    grid-row-gap: 1%;
+    padding-bottom: 3rem;
+}
+
+.main-content {
+    width: 70%;
+}
+
+@media screen and (max-width: 575px) {
+    .main-content {
+        width: 100%;
+    }
+}
+
+@media screen and (max-width: 575px) {
+    .dogs {
+        width: 100%;
+        grid-template-columns: repeat(auto-fill, minmax(0rem, 100%));
+        overflow: auto;
+    }
+}
+
+@media screen and (min-width: 576px) and (max-width: 1279px) {
+    .dogs {
+        width: 100%;
+        grid-template-columns: repeat(auto-fill, minmax(0rem, 45%));
+    }
+}
+
+.blog-preview {
+    width: 20%;
+    margin-left: 3rem;
+}
+
+.blog-preview img {
+    width: 100%;
+    margin-top: 2rem;
+}
+
+.blog-preview h2 {
+    padding-top: 3rem;
+    padding-bottom: 0rem;
+}
+
+.blog-preview h3 {
+    margin-top: 2rem;
+}
+
+.blog-preview p {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+@media screen and (max-width: 575px) {
+    .blog-preview {
+        display: none;
+
+    }
+}
+
+.blog {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: space-between;
+    margin-bottom: 3rem;
+
+}
+
+.blog img {
+    width: 100%;
+    height: 300px;
```

Comment:
This will make your blog images stretched on mobile. I would suggest setting a width, such as 20rem. Then set your height to auto. `height: auto;`